src/components/ReportActionItem/TripRoomPreview.tsx

```diff
@@ -0,0 +1,204 @@
+            isSmallAvatarSubscriptMenu
+        />
+    );
+}
+
+const renderItem = ({item}: {item: Reservation}) => <ReservationView reservation={item} />;
+
+function TripRoomPreview({action, chatReportID, containerStyles, contextMenuAnchor, isHovered = false, isWhisper = false, checkIfContextMenuActive = () => {}}: TripRoomPreviewProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const [chatReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${chatReportID}`);
+    const [iouReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${chatReport?.iouReportID}`);
+
+    const tripTransactions = ReportUtils.getTripTransactions(chatReport?.iouReportID, 'reportID');
+    const reservations: Reservation[] = TripReservationUtils.getReservationsFromTripTransactions(tripTransactions);
+    const dateInfo = chatReport?.tripData ? DateUtils.getFormattedDateRange(new Date(chatReport.tripData.startDate), new Date(chatReport.tripData.endDate)) : '';
+    const {totalDisplaySpend} = ReportUtils.getMoneyRequestSpendBreakdown(chatReport);
+
+    const displayAmount = useMemo(() => {
+        if (totalDisplaySpend) {
+            return CurrencyUtils.convertToDisplayString(totalDisplaySpend, iouReport?.currency);
+        }
+
+        // If iouReport is not available, get amount from the action message (Ex: "Domain20821's Workspace owes $33.00" or "paid ₫60" or "paid -₫60 elsewhere")
+        let displayAmountValue = '';
+        const actionMessage = action.message?.[0]?.text ?? '';
+        const splits = actionMessage.split(' ');
+
+        splits.forEach((split) => {
+            if (!/\d/.test(split)) {
+                return;
+            }
+
+            displayAmountValue = split;
+        });
+
+        return displayAmountValue;
+    }, [action.message, iouReport?.currency, totalDisplaySpend]);
+
+    return (
+        <OfflineWithFeedback
+            pendingAction={action?.pendingAction}
+            shouldDisableOpacity={!!(action.pendingAction ?? action.isOptimisticAction)}
+            needsOffscreenAlphaCompositing
+        >
+            <View style={[styles.chatItemMessage, containerStyles]}>
+                <PressableWithoutFeedback
+                    onPressIn={() => DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
+                    onPressOut={() => ControlSelection.unblock()}
+                    onLongPress={(event) => showContextMenuForReport(event, contextMenuAnchor, chatReportID, action, checkIfContextMenuActive)}
+                    shouldUseHapticsOnLongPress
+                    style={[styles.flexRow, styles.justifyContentBetween, styles.reportPreviewBox, styles.cursorDefault]}
+                    role={CONST.ROLE.BUTTON}
+                    accessibilityLabel={translate('iou.viewDetails')}
+                >
+                    <View style={[styles.moneyRequestPreviewBox, styles.p4, styles.gap5, isHovered || isWhisper ? styles.reportPreviewBoxHoverBorder : undefined]}>
```

Comment:
Are we using the whisper pattern here? Could you show me the case for this?

Reply:
You're right, we don't have this case in the app, I removed that